images/keycloak/tests/01-docker-production-mode.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+#
+# Tests Keycloak in production mode. This requires a keystore to be created
+# and mounted into the Keycloak container image.
+#
+# Validation includes ensuring Keycloak is running, and that we can authenticate
+# with the API to retrieve a list of users.
+#
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+KEYCLOAK_HOSTNAME="localhost"
+KEYCLOAK_PORT=$FREE_PORT
+KEYCLOAK_URL="https://$KEYCLOAK_HOSTNAME:$KEYCLOAK_PORT"
+KEYSTORE_PATH="/tmp/server.keystore"
+KEYSTORE_PASSWORD="placeholder"
+
+# Define log entries we are looking for in the keycloak logs here.
+declare -a terms=(
+  "Listening on: https://0.0.0.0:8443"
+  "Profile prod activated"
+)
+
+declare -a missing_terms=()
+
+create_keystore() {  
+  rm -rf "$KEYSTORE_PATH"
+  keytool -v -keystore $KEYSTORE_PATH \
+  -alias $KEYCLOAK_HOSTNAME \
+  -genkeypair -sigalg SHA512withRSA -keyalg RSA \
+  -dname CN=$KEYCLOAK_HOSTNAME \
+  -storepass $KEYSTORE_PASSWORD
```

Comment:
This is really a nit, but I'd indent this a bit for better visualization
```suggestion
    -alias $KEYCLOAK_HOSTNAME \
    -genkeypair -sigalg SHA512withRSA -keyalg RSA \
    -dname CN=$KEYCLOAK_HOSTNAME \
    -storepass $KEYSTORE_PASSWORD
```

Reply:
100%, thanks and also for taking the time to do the inline suggestions